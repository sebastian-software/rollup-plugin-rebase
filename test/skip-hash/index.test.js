import { bundle, clean, list, read } from "../util"

const root = __dirname

test("Skip hash", async () => {
  await bundle(root, "index.js", "output/index.js", {
    skipHash: true,
  })

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()

  await clean(root, "output")
})
