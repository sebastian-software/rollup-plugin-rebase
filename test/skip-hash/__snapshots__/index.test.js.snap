// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skip hash 1`] = `
"content.md
image.png
index.js"
`;

exports[`Skip hash 2`] = `
"import content from './content.md';
import image from './image.png';

function index() {
  return content + image
}

export { index as default };
"
`;
